specification/portalservices/resource-manager/Microsoft.PortalServices/preview/2023-01-01-preview/extensions.json

```diff
@@ -34,6 +34,42 @@
     }
   },
   "paths": {
+    "/providers/Microsoft.PortalServices/operations": {
+      "get": {
+        "tags": [
+          "Operations"
+        ],
+        "description": "Lists all of the available Portal Services RP operations.",
+        "operationId": "Operations_List",
+        "x-ms-examples": {
+          "Compile single file using inline content": {
+            "$ref": "examples/OperationsList.json"
+          }
+        },
+        "parameters": [
+          {
+            "$ref": "../../../../../common-types/resource-management/v3/types.json#/parameters/ApiVersionParameter"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "OK - The request has succeeded.",
+            "schema": {
+              "$ref": "../../../../../common-types/resource-management/v3/types.json#/definitions/OperationListResult"
+            }
+          },
+          "default": {
+            "description": "Error response describing why the operation failed.",
+            "schema": {
+              "$ref": "../../../../../common-types/resource-management/v3/types.json#/definitions/ErrorResponse"
+            }
+          }
+        },
+        "x-ms-pageable": {
+          "nextLinkName": "nextLink"
+        }
+      }
+    },
     "/providers/Microsoft.PortalServices/compilefile": {
```

Comment:
this isn't a valid english word right, so how did it pass spellcheck?

Reply:
Please remove 'compilefile' from customwords.txt, and use camelCasing, if this is still a preview-only API

Reply:
I see its added in custom-words.txt but I doubt that is appropriate.

Reply:
This was reviewed in a previous PR.